src/sigstore.test.ts

```diff
@@ -24,9 +24,8 @@ describe('sign', () => {
   const payload = Buffer.from('Hello, world!');
 
   // Signer output
-  const signedPayload = {
-    base64Signature: 'signature',
-    cert: 'cert',
+  const bundle = {
```

Comment:
Thoughts on using a real example here and on line 77? e.g. https://github.com/sigstore/sigstore-js/pull/107/files#diff-03b4f24fce55af1e22db038dfd3ff8429227f5ca4d6b284b663628b6db4fac2aR27

Reply:
Yeah, good call. This will give me an opportunity to check that the return value is truly the `Bundle.toJSON` form of the bundle.